src/NUnitFramework/framework/Attributes/OrderAttribute.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NUnit.Framework.Interfaces;
+using NUnit.Framework.Internal;
+
+namespace NUnit.Framework
+{
+    public class OrderAttribute : NUnitAttribute, IImplyFixture, IApplyToTest
```

Comment:
I don't think this should imply a fixture. That would mean that any methods with an Order attribute, even if it's not a test method, would make the class a fixture.